requirements.txt

```diff
@@ -1,5 +1,5 @@
 six>=1.9.0
-requests>=2.7.0
```

Comment:
last version of `requests` conflicting with our pinned `chardet` package version.